docs/ci-services.md

```diff
@@ -0,0 +1,55 @@
+# CI Services
+
+The CI service blob defines the supported CI services the user will be
+presented with when running the scaffolder.
+
+## Example
+
+```javascript
+{
+  TravisCI: {scaffolder: travisScaffolder, public: true}
+}
+```
+
+## Config
+
+### keys: __string__
+
+Keys in the CI services blob define the options the user will be presented with
```

Comment:
```suggestion
Keys in the CI services map? define the choices the user will be presented with
```

may also be worth mentioning that the keys are user defined and don't impact any programatic logic (assuming that is true. i think it is for ci services, but may not be in some other choice situations. i havent thought of great solutions when that is the case)